src/components/HistoryPanel.tsx

```diff
@@ -72,23 +88,41 @@ export default function VotePanel(_props: Props) {
       q,
       outcomeByHandle.get(q.questionHandle),
     ]);
-  const receivedItems = questionsWithAnswers.map(([qData, aData], index) => (
-    <motion.div
-      key={index}
-      variants={cardVariant}
-      className="rounded-lg border-gray border shadow-md mb-4"
-    >
-      <QuestionDetails
-        details={qData}
-        outcome={aData?.question === qData.questionHandle ? aData : undefined}
-        instance={instance}
-      />
-    </motion.div>
-  ));
+  const receivedItems =
```

Comment:
let's try to avoid nested ternaries. (surprised we don't have a lint rule)

would this work?
```
const receivedItems = dataLoaded ?
  instanceStatus === 'received' && questionsWithAnswers.map(
  )
  : (
    <div className="italic text-center mt-16">No questions yet.</div>
  );
```